README.rst

```diff
@@ -600,6 +600,13 @@ The reasons are:
   containing `document.write()` calls) can add extra tokens, so the parsing
   process actually modifies the input.
 
+To dive deeper into the parsing algorithm, It's essential to understand how
```

Comment:
> It
should be lowercase

Reply:
Fixed this and I appreciate it. Thank you very much.